django/db/__init__.py

```diff
@@ -3,14 +3,14 @@
     DEFAULT_DB_ALIAS, DJANGO_VERSION_PICKLE_KEY, ConnectionHandler,
     ConnectionRouter, DatabaseError, DataError, Error, IntegrityError,
     InterfaceError, InternalError, NotSupportedError, OperationalError,
-    ProgrammingError,
+    ProgrammingError, digest,
```

Comment:
It can be used [here](https://github.com/django/django/blob/master/django/db/backends/base/schema.py#L118) as well, hence the import.